### DISCUSSION TITLE
Using the 'Tailwind' name in a WordPress plugin?

### DISCUSSION THREAD

**Discussion by dgwyer** [tailwindlabs/tailwindcss#2736] [Help] [answered]
I'm currently experimenting with a WordPress plugin to allow you to add Tailwind classes to HTML directly in the block editor and have it preview the results.

However, before I spend too much time on this project,  I was wondering if there would be an issue with using 'Tailwind' in the plugin name? I know the Tailwind CSS license is MIT but I'm not sure if this covers name usage. The WordPress plugin itself will be GPL, and there would likely be a Pro version for advanced usage but this will again be GPL.

Is anyone able to shed any light on this. I'd love to get a fully working prototype added to the WordPress plugin directory but I'd need confirmation it was OK first as this will likely get flagged upon plugin submission.

**Comment by adamwathan** [comment, marked as answer]
Hey @dgwyer! Check out our Brand Guidelines policy for some info:

https://tailwindcss.com/brand

In short "Tailwind Preview" is against the guidelines, but something like "Class Preview for Tailwind CSS" is okay.

**Comment by dgwyer** [comment]
Thanks @adamwathan for clarifying!
